### PR TITLE
docs: add documentation for transformed data section and variables

### PR DESCRIPTION
**Pull request by florianesser**
Extend documentation for the [Transformed data](https://help.wetransform.to/docs/references/data/2018-03-06-reference-data-transformed-data) section of a dataset. Add documentation related to project variables and the use of placeholders to pass special values, e.g. `{{dataset.id}}`.

ING-4530

### CODE REVIEW

**Review by morch23mj — APPROVED** [review]
LGTM.

File: docs/references/data/2018-03-06-reference-data-transformed-data.md [hunk 1]
```diff
@@ -11,12 +11,12 @@ sidebar_position: 8
 
 The Transformed data section displays transformation projects associated with this data set. Click the project name or the chevron to view general properties and the current status of the transformation.
 
-Under Project, click the transformation project name to edit project properties.
+<img src={require("/images/help/en/dataset_transformed_data.png").default} alt="" title="The Transformed data section " className="img-responsive img-inline-help"/>
 
-Under Target data set, click the data set name to edit data set properties. Click the arrow button to download the transformed data set.
+The page provides links to the transformation project and to the transformed dataset ("Target dataset"). The result of the last transformation run is shown near the bottom, together with a link to show the log output of the transformation run. "The Transformed types" table lists the number of occurrences of each type in the source data and the number of occurrences of each type in the target data.
```
Comment (JohannaOtt) [inline docs/references/data/2018-03-06-reference-data-transformed-data.md:16]: Would it make sense to add an information on that is sometimes not working as expected? See https://wetransform.atlassian.net/browse/ING-1693 and https://wetransform.atlassian.net/browse/ING-1693

File: docs/references/data/2018-03-06-reference-data-transformed-data.md [hunk 2]
```diff
@@ -11,12 +11,12 @@ sidebar_position: 8
...
 
-Under Last transformation result, you can view the status of the last transformation, the time it started, the time it finished and how long the transformation took to execute.
+In case the transformation project contains variables, the "Project variables:" section gives access to these. The values of the variables can be changed here so that subsequent transformation runs (which can be triggered via the "Start transformation" button) will use the new values.
```
Comment (JohannaOtt) [inline docs/references/data/2018-03-06-reference-data-transformed-data.md:18]: Same here - see https://wetransform.atlassian.net/browse/ING-3520

File: docs/references/data/2018-03-06-reference-data-transformed-data.md [hunk 3]
```diff
@@ -11,12 +11,12 @@ sidebar_position: 8
...
 
-Click the Log output link to view a summary of transformation process steps.
+By clicking the button next to the input fields, an Autofill assistant can be opened. It can be used to insert special placeholders into the project variable that will be replaced by their specific value before being passed to the transformation engine. For example can the placeholder `{{dataset.name}}` be used to pass the name of the dataset to the transformation engine.
 
-The Transformed types table lists the number of occurrences of each type in the source data and the number of occurrences of each type in the target data.
+Another practical example for this feature is to pass the hale»connect dataset ID placholder `{{dataset.id}}` to the variable `DATASET_ID` of a XPlanung to INSPIRE transformation which is used to generate unique identifiers for `OfficialDocumentation` elements if the project variable `FORCE_UNIQUE_OFFICIALDOCUMENTATION_ID` is set to `true`.
```
Comment (JohannaOtt) [inline docs/references/data/2018-03-06-reference-data-transformed-data.md:22]: placholder --> placeholder

File: i18n/de/docusaurus-plugin-content-docs/current/references/data/2018-03-06-reference-data-transformed-data.md [hunk 4]
```diff
@@ -9,14 +9,14 @@ sidebar_position: 8
 
 ---
 
-Im &laquo;Abgeleitete Daten&raquo;-Bereich werden die Transformations-Projekte angezeigt, die mit dem ausgewählten Datensatz verbunden sind. Klicken sie auf den Namen des Transformations-Projektes oder das Pfeilsymbol um grundlegende Eigenschaften und den aktuellen Status der Transformation angezeigt zu bekommen.
+Der Bereich Transformierte Daten zeigt die Transformationsprojekte an, die mit diesem Datensatz verbunden sind. Klicken Sie auf den Projektnamen, um allgemeine Eigenschaften und den aktuellen Status der Transformation anzuzeigen.
```
Comment (JohannaOtt) [inline i18n/de/docusaurus-plugin-content-docs/current/references/data/2018-03-06-reference-data-transformed-data.md:12]: Sie --> sie

There is an agreement to write sie with a lower case s in the documentation.

File: i18n/de/docusaurus-plugin-content-docs/current/references/data/2018-03-06-reference-data-transformed-data.md [hunk 5]
```diff
@@ -9,14 +9,14 @@ sidebar_position: 8
...
 
-Klicken sie unter &laquo;Projekt&raquo; auf den Namen des Transformations-Projektes, um die Eigenschaften des Projektes zu bearbeiten.
+<img src={require("/images/help/de/dataset_transformed_data.png").default} alt="" title="The Transformed data section " className="img-responsive img-inline-help"/>
```
Comment (JohannaOtt) [inline i18n/de/docusaurus-plugin-content-docs/current/references/data/2018-03-06-reference-data-transformed-data.md:14]: title of the image to be translated too?

**Review by JohannaOtt — COMMENTED** [review]
Sorry, only getting around reviewing this now. Still added by comments.

**Pull request merged**